readme.md

```diff
@@ -4,6 +4,8 @@
 
 [![NPM](https://nodei.co/npm/monument.png?downloadRank=true&stars=true)](https://nodei.co/npm/monument/)
 
+[![npm version](https://img.shields.io/npm/v/monument.svg)](https://www.npmjs.com/package/monument)
```

Comment:
can you match the flat-square style of the other buttons? Other then that looks fantastic!